stable-diffusion/anaconda-project.yml

```diff
@@ -0,0 +1,120 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+
+# required: name MUST match the directory name.
+#           name must be of the form r'[a-z_]+'
+name: stable-diffusion
+# required: short project description, used in the main gallery
+description: An example panel app for stable diffusion
+
+# required: config 
+examples_config:
+  #### REQUIRED ###
+  # Creation date following the YYYY-MM-DD format
+  created: 2022-01-30
+  # List of the GITHUB handle of the authors/maintainers
+  maintainers:
+  - "sandhujasmine"
+  # List of labels displayed in the project card
+  # Each label must be a name (e.g. panel) that
+  # refers to a SVG badge located in doc/_static/labels
+  # (e.g. doc/_static/labels/panel.svg)
+  labels:
+  - "channel_conda-forge"
+  - "panel"
+
+  ### OPTIONAL ###
+  # Listed deployments will by default automatically be started.
+  # Maximum number of deployments is 2.
+  # Each deployment must declare the command it deploys, options
+  # include "notebook" or "dashboard".
+  deployments:
```

Comment:
I assume this project will have no deployment on AE5, you can remove all of that.

Reply:
To deploy on AE5, we would need to add a GPU node with a resource profile and point this deployment to use that resource profile. 

I'll remove this section for now but adding this to the TODO list above. It would be nice.